src/biome/text/api_new/pipeline.py

```diff
@@ -497,6 +497,23 @@ def type_name(self) -> str:
         """The pipeline name. Equivalent to task head name"""
         return self.head.__class__.__name__
 
+      
+    def _filter_parameters_by_predicate(
+        self, predicate: Callable[[], bool]
+    ) -> List[str]:
+        filtered_parameters = filter(predicate, self._model.named_parameters())
+        return [name for name, p in filtered_parameters]
+
+    @property
+    def trainable_parameters(self) -> List[str]:
+        """Returns the name of pipeline trainable parameters"""
+        return self._filter_parameters_by_predicate(lambda p: p.requires_grad)
+
+    @property
+    def frozen_parameters(self) -> List[str]:
+        """Returns the number of frozen parameters of pipeline"""
+        return self._filter_parameters_by_predicate(lambda p: not p.requires_grad)
```

Comment:
Does `frozen_parameters` return the **number** and `trainable_parameters` the **name** of the parameters? Why not numbers for both?

Reply:
We just keep info about trainable params